pandas/tests/dtypes/test_inference.py

```diff
@@ -577,6 +577,28 @@ def test_integers(self):
         result = lib.infer_dtype(arr, skipna=True)
         assert result == "integer"
 
+    @pytest.mark.parametrize(
+        "arr, skipna, expected",
```

Comment:
You don't need to parameterize `expected` here.  It's `integer-na` when `skipa` is `False` and `integer` when `skipna` is `True`.

Reply:
Ok, sure, updated